docs/index.md

```diff
@@ -11,7 +11,7 @@ Releases of the specification have been formally published through HL7.
 
 Ballot versions are the precursors to a released version and go through a period of community feedback. They are archived here for historical posterity.
 
-Hooks that have reached maturity level 4 or above (as described by the [Hook Maturity Model](1.0/#hook-maturity-model)) are published with the specification here. Proposed hooks (level 3 or below, are found at [https://cds-hooks.org](https://cds-hooks.org))
+Hook definitions are published with the specification here in the hooks topic. Note that hook definitions are not part of the CDS Hooks specification, they are independent specifications that follow the [Hook Maturity Model](1.0/#hook-maturity-model). Proposed hooks not yet available here can be found at [https://cds-hooks.org](https://cds-hooks.org).
```

Comment:
@brynrhodes - If we used wording like this, we could make this change at cds-hooks.org as well. 
```suggestion
Hook definitions are published with the specification. Note that hook definitions are not part of the CDS Hooks specification, they are independent specifications that follow the [Hook Maturity Model](1.0/#hook-maturity-model). Hooks are published by HL7. Proposed hooks can be found at [https://cds-hooks.org](https://cds-hooks.org).
```